src/renderer/marketplace/incorporation/details/incorporations-details-container.jsx

```diff
@@ -9,15 +9,22 @@ import { walletSelectors } from 'common/wallet';
 import { withStyles } from '@material-ui/core/styles';
 import { incorporationsSelectors, incorporationsOperations } from 'common/incorporations';
 import { IncorporationsDetailsPage } from './incorporations-details-page';
+import config from 'common/config';
 
 const styles = theme => ({});
 
 class IncorporationsDetailsContainer extends MarketplaceIncorporationsComponent {
 	state = {
 		tab: 'description',
-		loading: false
+		loading: false,
+		cryptoValue: 0
 	};
 
+	constructor() {
+		super();
+		this.handleCryptoValueChange = this.handleCryptoValueChange.bind(this);
```

Comment:
this is not needed becaus `handleCryptoValueChange` is an arrow function and is bound already